network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,75 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-RTT-1.5"]
+==== Round-trip time (RTT)
+In 1.5, you can use TCP handshake Round-Trip Time (RTT) captured from the `fentry/tcp_rcv_established` eBPF hookpoint to read SRTT to analyze network flows. In the *Overview*, *Network Traffic*, and *Topology* pages in web console, you can monitor network traffic and troubleshoot with RTT metrics, filtering, and edge labeling. For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-RTT-overview_nw-observe-network-traffic[RTT Overview] and xref:../network_observability/observing-network-traffic.adoc#network-observability-RTT_nw-observe-network-traffic[Working with RTT].
+
+[id="network-observability-metrics-dashboard-enhancements"]
+==== Metrics, dashboards, and alerts enhancements
+The Network Observability metrics dashboards in *Observe* → *Dashboards* → *NetObserv* has new metrics types you can use to create Prometheus alerts. The list items are now in the `includeList` whereas before they were in `ignoreTags`. For a complete list of these metrics, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-improved-lokistack-integration"]
+==== Improvements for Network Observability without Loki
+You can create Prometheus alerts for the *Netobserv* dashboard using DNS, Packet drop, and RTT metrics, even if you don't use Loki. In 1.5, these metrics were only available for querying and analysis in the *Network Traffic*, *Overview*, and *Topology* views, which are not available without Loki. For more information, see xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-metrics_metrics-dashboards-alerts[Network Observability Metrics].
+
+[id="network-observability-zones"]
+==== Availability zones
+You can configure the `FlowCollector` to collect information about the cluster availability zones. This allows you to enrich network flow data with the link:https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone[`topology.kubernetes.io/zone`] label value applied to the nodes.  
+
+// Commenting this out until https://github.com/openshift/openshift-docs/pull/70974/files#top is merged--- For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-working-with-zones[Working with availability zones].
+
+[id="network-observability-enhanced-configuration-and-ui-1.5"]
+==== Notable enhancements 
+
+[discrete]
+[id="web-console-enhancements-1.5_{context}"]
+===== Web console enhancements:
+
+* There are new panels added to the *Overview* view of the Network Traffic.
+* There is a new panel display option to focus on one panel while keeping others viewable but smaller focus.
```

Comment:
There are actually a bunch of UI improvements here. Do we want to list them all ?

See:
https://issues.redhat.com/browse/NETOBSERV-657
https://issues.redhat.com/browse/NETOBSERV-676

Such as:
- Collection latency warning in time range selection popup
- Panels & columns popups quick filters
- New graphs available for DNS & RTT: Min, Max, P90, P99
- Switch graph type
- Show Top / Overall

Reply:
Yeah I guess we don't necessarily document all of these improvements, but I agree we can point them all out.